headers/headers.go

```diff
@@ -0,0 +1,11 @@
+package headers
+
+const (
+	XResolution = "ResX"
+	YResolution = "ResY"
+	FPS         = "FPS"
+	Model       = "Model"
+	Brand       = "Brand"
+	PixelBits   = "PixelBits"
+	FrameSize   = "FrameSize"
```

Comment:
From our skype call I had thought that you were using the CPTV field character in the headers. This is better.

Reply:
Oh ye but i changed it to use this